test/ts-extension-eslint.config.mjs

```diff
@@ -0,0 +1,8 @@
+import neostandard, { plugins } from '../index.js'
+
+export default [
+  ...neostandard({
+    ts: true,
+  }),
+  ...plugins['typescript-eslint'].configs.recommended,
```

Comment:
I've added a similar test locally: 

```ts
 ...plugins['typescript-eslint'].config(
    ...plugins['typescript-eslint'].configs.recommended,
    ...plugins['typescript-eslint'].configs.strict
  )
```

just to be sure to test the helper with several configurations.

Reply:
Should we add such a variant here as well do you think?

Reply:
It depends if the helper is actually triggering the error: on poolifier, if I comment out 

 ...tseslint.config(...) 

there's no error.